fiatconnect-api.md

```diff
@@ -1252,7 +1256,8 @@ return an HTTP `400` error code, along with the following response body:
 
 ###### 3.3.4.1.2.3. HTTP `404`
 
-If the selected fiat account does not exist, the server MUST respond with an HTTP `404` error code, along with the following response body:
+If the selected fiat account does not exist, or if a quote with the quote ID given in the request does not exist,
```

Comment:
The equivalent `HTTP 404` section under `/transfer/out` hasn't been updated to match this; if the intention is to require quote IDs for both transfers in and transfer out, you should update this section there as well.

Reply:
I updated to use `InvalidQuote` errors for both missing and expired quotes, so providers could delete expired quotes whenever they want